CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### 🚨 Breaking Changes
 
-None
+- The handling of `device` and `dtype` was overhauled. They might not behave as expected. `Element`s also no longer have a `device` attribute. (see #115) (@jank324)
 
```

Comment:
I think it's worth mentioning explicitly in the documents that it actually changes the behaviour of element creation, i.e. one can now create elements conveniently as before `v.6.0`, without the requirement to wrap every parameter as tensor first

```python
dipole = cheetah.Dipole(length=0.1, angle=0.1, dtype=torch.float32)
```

Reply:
I think you are right, but I also think this should be tested before making that claim.